packages/dashboard/src/components/Card/examples/Card.Chart.MultiStackedBar.Example.tsx

```diff
@@ -46,13 +46,15 @@ export class MultiStackedBarChartExample extends React.Component<{}, {}> {
       chartLabels: ['Monitored', 'Unmonitored'],
       chartType: ChartType.StackedBarChart,
       data: points,
+      chartUpdatedOn: 'Updated 6:20 pm today',
       compactChartWidth: 394,
       legendColors: colors
     };
 
     const textContent = {
       subHeaderText: 'Edit data loss prevention policies',
-      bodyText: 'Information based on your data loss prevention (DLP) policies'
+      bodyText:
+        'Information based on your data loss prevention (DLP) policies may take upto 24 hours to appear in this activity report'
```

Comment:
Super nit but "up to" is grammatically correct vs. "upto"

Reply:
Updated :)